exporters/tests/test_pelorus_base_plugin.py

```diff
@@ -316,7 +315,9 @@ async def test_proper_receive_metric(json_payload):
         "webhook.plugins.pelorus_handler_base.PelorusWebhookPlugin._receive",
         new_callable=AsyncMock,
     ) as mock_receive:
-        mock_receive.return_value = json.loads(json_payload)
```

Comment:
No need to change here, its commit metric, not deploy

(create test specific for deploy?)

Reply:
actually it will be valid to only accept timestamps from the time of starting the application. I know that at the moment it's not really valid, but really it does not change the test logic, just modifying the time to be current and not in the past, so I think it's not harmful change here.